src/components/offers/details/OffersDetails.js

```diff
@@ -19,6 +19,8 @@ export const OfferDetails = (props) => {
         history.push(`/offers/apply/${offer.id}`)
     }
 
+    console.log(offer)
```

Comment:
Potrzebujemy tutaj loga?

Reply:
Nie, wiedziałem, że gdzieś jest, ale nie mogłem go znaleźć, to stwierdziłem, że znajdziesz, bo je zawsze znajdujesz XDDD